statistics/histogram.go

```diff
@@ -439,6 +455,9 @@ func (hg *Histogram) SplitRange(ranges []*ranger.NewRange) []*ranger.NewRange {
 				HighExclude: false})
 			ranges[0].LowVal[0] = upper
 			ranges[0].LowExclude = true
```

Comment:
Can we enforce LowExclude to false?
By PrefixNext Bytes or increase interger?

Reply:
Could it possible that increase integer cause overflow?